### PR TITLE
[CROM-6661][BT-114] Reference Disk Opt-in: UI work 

### PR DESCRIPTION
**Pull request by mcovarr**
Confirmed with Chrome network traffic monitoring that the `useReferenceDisks` parameter was being sent in submission requests as expected.

### CODE REVIEW

**Review by panentheos — APPROVED** [review]
Looks good. Tested locally by verifying the flag in the network call.

File: src/pages/workspaces/workspace/workflows/WorkflowView.js [hunk 1]
```diff
@@ -763,6 +764,13 @@ const WorkflowView = _.flow(
                 href: 'https://support.terra.bio/hc/en-us/articles/360039681632',
                 ...Utils.newTabLinkProps
               }, ['Click here to learn more.'])
+            ]),
+            span({ style: { margin: '0 0.5rem 0 1rem' } }, [
+              h(LabeledCheckbox, {
+                checked: useReferenceDisks,
+                onChange: v => this.setState({ useReferenceDisks: v }),
+                style: { marginLeft: '1rem' }
+              }, [' Use reference disks'])
```
Comment (kshakir) [inline src/pages/workspaces/workspace/workflows/WorkflowView.js:804]: Is that space before `Use` supposed to be there?

**Review by grsterin — APPROVED** [review]
Looks good, once Khalid's comment is answered

File: src/pages/workspaces/workspace/workflows/WorkflowView.js (hunk 1 above)
Comment (panentheos) [reply src/pages/workspaces/workspace/workflows/WorkflowView.js:804]: Yes, the `LabeledCheckbox` component puts the checkbox inline with the text, so we typically put a space to create a small gap between them.

**Comment by mcovarr** [comment]
Added info box with link to Terra Support article. I also had to do a nontrivial rebase, confirmed the 'useReferenceDisks' parameter is still working per network traffic.

**Review by kshakir — APPROVED** [review]

**Review by grsterin — APPROVED** [review]

**Review by breilly2 — APPROVED** [review]
👍  Looks good. I tested locally by verifying that the flag shows up in the call to Rawls, both on initial submission and when rerunning failures.

**Comment by mcovarr** [comment]
Good news: I fixed the integration test issues with a rebase.

Less good news: Said rebase was not completely trivial. Also in the process of testing I noticed that 'Use Reference Disks' was missing from `SubmissionDetails.js` so I added that in. All of which means I must ask for another round of review... 😞

**Review by grsterin — APPROVED** [review]

**Review by breilly2 — APPROVED** [review]
👍   Still looks good. I verified that "Use Reference Disks" shows up on the job details screen.

BTW, the test failure was more likely fixed by a service account clean-up I did yesterday. See this slack thread: https://broadinstitute.slack.com/archives/C01EHNUM73R/p1612885718027100

**Pull request merged**